pkg/operation/botanist/controlplane.go

```diff
@@ -299,8 +292,51 @@ func (b *Botanist) HibernateControlPlane(ctx context.Context) error {
 		}
 	}
 
+	return client.IgnoreNotFound(b.ScaleETCDToZero(ctx))
+}
+
+// ScaleETCDToZero scales ETCD main and events to zero
+func (b *Botanist) ScaleETCDToZero(ctx context.Context) error {
 	for _, etcd := range []string{v1beta1constants.ETCDEvents, v1beta1constants.ETCDMain} {
-		if err := kubernetes.ScaleEtcd(ctx, c, kutil.Key(b.Shoot.SeedNamespace, etcd), 0); client.IgnoreNotFound(err) != nil {
+		if err := kubernetes.ScaleEtcd(ctx, b.K8sSeedClient.Client(), kutil.Key(b.Shoot.SeedNamespace, etcd), 0); client.IgnoreNotFound(err) != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// ScaleETCDUp scales ETCD main and events replicas to one
+func (b *Botanist) ScaleETCDUp(ctx context.Context) error {
```

Comment:
Maybe we should directly name this `ScaleETCDToOne` similar to `ScaleETCDToZero` and `ScaleGardenerResourceManagerToOne`